AppController/djinn.rb

```diff
@@ -4085,6 +4085,22 @@ def build_uncommitted_changes()
     build_java_appserver if status.include?('AppServer_Java')
   end
 
+  def configure_ejabberd_cert
+    # Replace APPSCALE_CONFIG_DIR/ejabberd.pem with private key and cert
+    # copied from tools. APPSCALE_CONFIG_DIR/ejabberd.pem is not the default
+    # ejabberd.pem file, so we are overwriting changes that we have made.
```

Comment:
@obino we're overwriting what we wrote into /etc/appscale/ejabberd.pem ... if that makes sense... but we are not overwriting /etc/ejabberd/ejabberd.pem.